source/extensions/extensions_build_config.bzl

```
@@ -260,6 +260,12 @@ EXTENSIONS = {
     "envoy.tracers.skywalking":                         "//source/extensions/tracers/skywalking:config",
     "envoy.tracers.opentelemetry":                      "//source/extensions/tracers/opentelemetry:config",
 
+    #
+    # Sampler
```

```suggestion
    # OpenTelemetry tracer samplers
```
Maybe? So it's clear it applies to the OTel tracer